doc/sphinxext/gallery_order.py

```diff
@@ -36,17 +63,20 @@
     '../galleries/tutorials/provisional'
 ]
 
-plot_types_order = [
-    '../galleries/plot_types/basic',
-    '../galleries/plot_types/stats',
-    '../galleries/plot_types/arrays',
-    '../galleries/plot_types/unstructured',
-    '../galleries/plot_types/3D',
-    UNSORTED
-]
+# plot_types_order = [
+#     '../galleries/plot_types/basic',
+#     '../galleries/plot_types/stats',
+#     '../galleries/plot_types/arrays',
+#     '../galleries/plot_types/unstructured',
+#     '../galleries/plot_types/3D',
+#     UNSORTED
+# ]
 
-folder_lists = [examples_order, tutorials_order, plot_types_order]
 
+plot_types_directory = "../galleries/plot_types/"
+plot_types_order = get_ordering(plot_types_directory, include_directory_path=True)
```

Comment:
Yes it does, because other folders are also structured in the same way as `plot_types` directory. Except for the `users_explain` directory, because its not included in `explicit_order_folders` but some of it's files are in `list_all`. I'm not sure why `users_explain` directory is not included in `explicit_order_folders` but because of this, i think its currently being sorted according to default values but that doesn't seem to totally line up with my understanding. because the sort keys returned by `MplExplicitOrder` will be UNSORTED thus defaulting to alphabetic ordering or LOC based ordering. But that doesn't seem to be the case when i'm going through the `user guide` section in doc website. Any clarification on how user_explain is sorted will help me greatly. 

Other than that, rest of the folders will work fine with my current implementation as far as i understand. I'll push an update soon with the modifications for other other folders.